rest/pom.xml

```diff
@@ -202,13 +202,12 @@
     </plugins>
   </build>
 
-  <profiles>    
+  <profiles>
      <profile>
         <id>auth</id>
         <activation>
             <property>
                 <name>auth</name>
```

Comment:
Shouldn't be the profile activation managed by -P flag? Do we need a SystemProperty -Dauth somewhere else? If yes, can we set it in the profile?